backend/lib/routes/members.js

```diff
@@ -55,6 +55,17 @@ router.route('/:name')
       next(err)
     }
   })
+  .put(async (req, res, next) => {
+    try {
+      const user = req.user
+      const namespace = req.params.namespace
+      const name = req.params.name
+      const roles = req.body.roles
+      res.send(await members.update({ user, namespace, name, roles }))
```

Comment:
```suggestion
      const body = req.body
      res.send(await members.update({ user, namespace, name, body }))
```
The route should not know about the body structure

Reply:
done